case-lib/lib.sh

```diff
@@ -296,9 +296,15 @@ fake_kern_error()
 get_ldc_subdir()
 {
     local subdir='intel/sof' # default
-    if test -e /sys/module/snd_sof_pci/parameters/fw_path; then
-        local fw_path
-        fw_path=$(cat /sys/module/snd_sof_pci/parameters/fw_path)
+    local fw_path
+    local fw_path_info='/sys/kernel/debug/sof/fw_path'
+
+    if test ! -e $fw_path_info; then
+	fw_path_info='/sys/module/snd_sof_pci/parameters/fw_path'
+    fi
+
+    if fw_path=$(cat $fw_path_info); then
```

Comment:
This commit adds `if ... ; then` but no `fi` termination so I don't see how this can work?

Reply:
Never mind, now I see what happened.